### ISSUE TITLE
LDAP Authentication

### ISSUE THREAD

**Issue by robodude666** [swarmpit/swarmpit#329]
FEATURE REQUEST
--------------------

**Description**

It would be great to see authentication supported via LDAP/Active Directory.

**Comment by adolfombarros** [comment]
looking forward to this feature

**Comment by wil-m** [comment]
Agree, it would be great to have this feature. It is show stopper for our company, so we had to start to use Portanier, even our development team likes Swarmpit much more.

**Comment by sgutermann** [comment]
The LDAP part never made it onto the Roadmap as far as I can tell. What would need to happen to get this onto the Roadmap?

**Comment by nohaapav** [comment]
> The LDAP part never made it onto the Roadmap as far as I can tell. What would need to happen to get this onto the Roadmap?

Probably someone with capacity :) Myself & @mrq1911 are pretty busy with our current agenda, but looks like there is plenty of people asking for the feature so i we might have a look. However can't promise delivery date.
